src/plugins/blocksmith/components/gallery-block-modal/index.jsx

```diff
@@ -64,6 +64,7 @@ class GalleryBlockModal extends React.Component {
       selectedGallery: gallery || null,
       selectedImage: image || null,
       startsAtPoster: block.get('start_at_poster'),
+      title: block.get('title'),
```

Comment:
@rsumilang want the backup of `''` here to avoid this warning

```
Warning: `value` prop on `input` should not be null. Consider using an empty string to clear the component or `undefined` for uncontrolled components.
```